app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,73 @@
+class RentalsController < ApplicationController
+  
```

Comment:
It might be useful to move some of the business logic from this controller into model methods.